src/App.js

```diff
@@ -40,4 +41,6 @@ const App = () => (
 
 App.displayName = 'App';
 
+initializeiOSSafariAutoScrollback();
+
```

Comment:
If this is something that's only intended to run on web, maybe you should consider importing and calling it only inside `src/setup/platformSetup/index.website.js` 

I'll would even go as far as 

```js
export default function () {
  // AppRegistry.runApplication(
  // ...

    if (getBrowser() === CONST.BROWSER.SAFARI || Str.contains(userAgent, 'iphone os 1')) {
        const applyScrollFix = require('../libs/iOSSafariAutoScrollback');
        applyScrollFix();
    }
```

This way extra code is imported only when needed

Reply:
@kidroca it is merged already... but that's a GREAT suggestion!

Reply:
Great suggestion @kidroca! If either of you wouldn't mind submitting a quick PR, I'll review it.

Reply:
Thanks guys! 
@roryabraham I'll open a PR about this today, when I get the chance